.docksal/addons/phpunit/phpunit

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+#: exec_target = cli
+
+## Runs PHPUnit tests found in custom code
+##
+## Usage: fin phpunit <args>
+##
+## Run `fin phpunit -h` for a full list of options.
+##
+## Runs with settings in the root `phpunit.xml` file.
+
+DOCROOT_PATH="${PROJECT_ROOT}/${DOCROOT}"
+DRUPAL_CORE_PATH="${DOCROOT_PATH}/core"
```

Comment:
It doesn't look like this var is used anywhere?

Reply:
yup, those are leftover from the `phpunit` addon (which doesn't work here, and doesn't pre-configure phpunit.xml for you). Good catch, I'll remove those too.